### DISCUSSION TITLE
Cannot select next video if controls are hidden

### DISCUSSION THREAD

**Discussion by Dima-Astreyko** [vimeo/player.js#1018] [Bug reports]
### Expected Behavior
Can select next video after video ends if controls are hidden

### Actual Behavior

If player has `controls: false` option, user cannot select next video in playlist.
The root div element has the following styles, which prevent any user actions with the player. As a result, user cannot select next video
```
.player.hide-controls-mode {
    pointer-events: none;
}
```

### Steps to Reproduce

- open the page [JSFiddle](https://jsfiddle.net/desfar/6fsce7pa/3/)
- click `play video` button 
- click `seek to end` button
- try to select next video

**Comment by rkrishnan8594** [comment]
Thanks for reporting this issue, @Dima-Astreyko, and for providing clear reproduction steps. We have this bug ticketed internally for later prioritization.
